### PR TITLE
KUZ-480 Added method isActionAllowed to security

### PR DESCRIPTION
**Pull request by xbill82**
Followed the guidelines in https://confluence.kaliop.net/display/LABS/2016-05-24+-+Client-side+Policies
Just dropped in @ballinette's code snippet and all the tests are green :)

**Please let me know if you estimate the tests are exhaustive: I surely forgot some edge cases.**

Question: is the filename `methods.test.js` ok? I was wondering if I should put it into the `userMethods.test.js`.

### CODE REVIEW

File: src/security/kuzzleSecurity.js [hunk 1]
```diff
@@ -709,5 +709,55 @@ KuzzleSecurity.prototype.userFactory = function(id, content) {
...
+ * policies provided as the first argument. An action is defined as a couple of
+ * action and controller (mandatory), plus an index and a collection(optional).
+ *
+ * @param {object} policies - The rights policies associated to a user
+ *                            (see getMyPolicies and getUserPolicies).
+ * @param {string} controller - The controller to check the action onto.
+ * @param {string} action - The action to perform.
+ * @param {string} index - (optional) The name of index to perform the action onto.
+ * @param {string} collection - (optional) The name of the collection to perform the action onto.
+ */
```
Comment (benoitvidis) [inline src/security/kuzzleSecurity.js]: nitpicking: missing @returns jsdoc

**Comment by benoitvidis** [comment]
+1

File: src/security/kuzzleSecurity.js (hunk 1 above)
Comment (xbill82) [inline src/security/kuzzleSecurity.js]: :+1:

**Comment by xbill82** [comment]
Hmmm... I'm wondering whether it wouldn't be better to move the tests into `userMethods.test.js`...

**Comment by scottinet** [comment]
No I think your initial approach is the right one: the `KuzzleSecurity` tests have been split into users, profiles and roles to make things separated and clean, and the new `isActionAllowed` doesn't fit in any of these files.

**Comment by scottinet** [comment]
+1
